README.rst

```diff
@@ -1780,6 +1780,17 @@ Choose from the list of available rules:
   - ``strings_containing_single_quote_chars`` (``bool``): whether to fix
     double-quoted strings that contains single-quotes; defaults to ``false``
 
+* **single_space_after_construct**
+
+  Ensures a single space after language constructs.
+
+  Configuration options:
+
+  - ``constructs`` (a subset of ``['break', 'continue', 'echo', 'goto', 'new',
+    'print', 'return', 'throw', 'yield', 'yield_from']``): list of constructs
+    which must be followed by a single space; defaults to ``['break',
+    'continue', 'goto', 'new', 'return', 'throw', 'yield']``
```

Comment:
Why is `yield` default, but `yield from` not?

Reply:
@dmvdbrugge 

Currently failing because of support for PHP 5.6 - not sure yet how to work around it.

Reply:
By only adding them if the version allows? There are more fixers with version-specific configuration.